### DISCUSSION TITLE
Updating OsmAnd~ via F-Droid repo in Obtanium

### DISCUSSION THREAD

**Discussion by SanskritFritz** [ImranR98/Obtainium#1580] [Q&A] [answered]
Hello I installed OsmAnd first from Neo Store and then I discovered Obtanium and fell in love. I transferred OsmAnd to Obtanium via the FDroid [url](https://www.f-droid.org/packages/net.osmand.plus). Now a newer OsmAnd version came out (4.7.10). When I try to update it in Obtanium, I get 3 release assets: net.osmand.plus_471003.apk; net.osmand.plus_471002.apk; net.osmand.plus_471001.apk. Which one should I use and how to decide that?
Thanks for Obtanium I now maintain 87 apps in it, and it works well!

**Comment by SanskritFritz** [comment, marked as answer]
Ah I just found out those assets correspond the different CPU architectures (armeabi-v7a, x86_64, arm64-v8a).

**Reply by WilliamKimball0** [reply]
> Ah I just found out those assets correspond the different CPU architectures (armeabi-v7a, x86_64, arm64-v8a).

Which of those was the arm64-v8a? The one ending in 1?

**Reply by SanskritFritz** [reply]
I listed the architectures in the order they are numbered in FDroid.
Look here: https://f-droid.org/packages/net.osmand.plus/
So, arm64-v8a is Version 4.9.10 (491003).
